api/helpers/ find-spectatable-games.js

```diff
@@ -6,13 +6,11 @@ module.exports = {
   description: 'Finds all the games that are available to spectate',
 
   fn: async (_, exits) => {
-
     const recentUpdateThreshhold = dayjs().subtract(5, 'minute').valueOf();
 
     try {
       const games = await Game.find({
-        status: false, // not open to join
-        result: -1, // not finished yet
+        status:  gameService.GameStatus.STARTED,
```

Comment:
I feel like there has to be a way that we can query the table to check the length of the array of players, I just haven't found it yet. I've tried stackoverflow and chatGPT but I've been unsuccesful so far, I'll keep trying to find a good solution for this because we do it in another spot as well.